lib/engine/util.go

```diff
@@ -408,6 +428,88 @@ func funcASBSRBp(fn func(string, bool, string) *bool) tengo.CallableFunc {
 	}
 }
 
+func funcASSISRIp(fn func(string, string, int, string) *int) tengo.CallableFunc {
```

Comment:
lol

Reply:
Yeah.... haha followed the naming convention that Tengo used for their stuff. These functions are simply functions that convert tengo type to golang types then call the func passed in. Everything after the `A` and before the `R` is the different argument types. Everything after the `R` is the return types. I'll add comments for each of them